glossary.ftd

```diff
@@ -1,22 +1,219 @@
 -- ft.page: Glossary
 
-For readers/writers of document by id, we have three kinds of id: `id: /foo/`,
-`full-id: /foo/-/x/`, `filename: foo.ftd`.
+An FPM Glossary
 
 
-Since we need to get `id`, `full-id` and `filename` of current document, to power
-document-readers, lets create processors `document-id`, `document-full-id` and
-`document-filename` as helpers as well. These three processors will return ids
-for current document.
+
+-- ft.h2: client project
```

Comment:
Not fond of `client project` term. It should be called a `clone`. You use `fpm clone` to create a `clone`. And from `clone` you run the `fpm sync` command, etc.